### ISSUE TITLE
[Bug][Combobox] The `suggest` prop prevents screen reader to read the selection

### ISSUE THREAD

**Issue by vveesseelliinnaa** [telerik/kendo-vue#475]
**Describe the bug**
When suggest prop is added screen reader doesn't read out selection

**To Reproduce**
1. Open this demo with NVDA - https://www.telerik.com/kendo-vue-ui/components/dropdowns/combobox/suggestions/
2. Navigate through the items using the arrow keys
3. Change the current selection by pressing Enter

Result: Nothing is read by the screen reader

**Expected behavior**
The screen reader should read every highlighted or selected value.

**Additional context**
Reported in Ticket ID: 1631071
